src/command_test.go

```diff
@@ -4,73 +4,132 @@ import (
-		{"serialize command", singleLineCommand, singleLineCommandString},
-		{"serialize multiline command", multiLineCommand, multiLineCommandString},
+		{"return command", singleLineCommand, singleLineCommandString},
+		{"escape multiline command", multiLineCommand, multiLineCommandString},
 	}
 
 	for _, tt := range table {
-		t.Run("should"+tt.testName, func(t *testing.T) {
-			result := tt.input.String()
-			if result != tt.expected {
-				t.Errorf("'%s' does not equal '%s'", result, tt.expected)
-			}
-		})
+		t.Run(
+			"should"+tt.testName, func(t *testing.T) {
+				result := tt.input.String()
+				if result != tt.expected {
+					t.Errorf("'%s' does not equal '%s'", result, tt.expected)
+				}
+			},
+		)
 	}
 }
 
 func TestParseCommands(t *testing.T) {
 
-	singleLineCommandString := ": 1605292765;Oskar Jung;tag;fg"
-	singleLineCommand := NewCommand(1605292765, "Oskar Jung", []string{"tag"}, "fg")
-	multiLineCommandString := ": 1605290038;Oskar Jung;multiline;echo \"asd\\\nfgh\""
-	multiLineCommand := NewCommand(1605290038, "Oskar Jung", []string{"multiline"},
-		"echo \"asd\\\nfgh\"")
-	invalidCommandString := ": 1605292765;Oskar Jung;fg"
+	invalidCommand := ": 1605292765;Oskar Jung;fg"
+	invalidUUIDCommand := ": uuid;1605292765;Oskar Jung;tag;fg"
+	invalidTimestampCommand := ": 3766fa42-29d7-11eb-ac1d-3f49909e858b;160529276b;Oskar Jung;tag;fg"
 
 	table := []struct {
 		testName string
 		input    string
 		expected []Command
 	}{
 		{"parse empty file without error", "", nil},
-		{"parse command", singleLineCommandString, []Command{singleLineCommand}},
-		{"ignore invalid line", invalidCommandString, nil},
-		{"parse multiline command", multiLineCommandString, []Command{multiLineCommand}},
-		{"parse multiple commands",
-			strings.Join([]string{multiLineCommandString, invalidCommandString, singleLineCommandString}, "\n"),
-			[]Command{multiLineCommand, singleLineCommand}},
-		{"parse only commands that start with newline",
-			singleLineCommandString + " " + singleLineCommandString,
-			[]Command{NewCommand(1605292765, "Oskar Jung", []string{"tag"},
-				"fg "+singleLineCommandString)},
+		{"parse command", singleLineCommandSerialized, []Command{singleLineCommand}},
+		{"ignore invalid line", invalidCommand, nil},
+		{"parse multiline command", multiLineCommandSerialized, []Command{multiLineCommand}},
+		{
+			"parse multiple commands",
+			strings.Join(
+				[]string{
+					multiLineCommandSerialized,
+					invalidCommand,
+					singleLineCommandSerialized,
+				}, "\n",
+			),
+			[]Command{multiLineCommand, singleLineCommand},
+		},
+		{
+			"parse only commands that start with newline",
+			singleLineCommandSerialized + " " + singleLineCommandSerialized,
+			[]Command{
+				{
+					Id:      singleLineCommand.Id,
+					Time:    time.Unix(1605292765, 0),
+					Author:  "Oskar Jung",
+					Tags:    []string{"tag"},
+					Command: "fg " + singleLineCommandSerialized,
+				},
+			},
+		},
+		{
+			"parse commands containing a semicolon",
```

Comment:
Can you also test commands that contain a `:`?

Reply:
see https://github.com/tome-sh/tome/blob/tome-show-keybinding/src/command_test.go#L91